cns/restserver/restserver.go

```diff
@@ -95,6 +95,8 @@ type EndpointInfo struct {
 	PodName       string
 	PodNamespace  string
 	IfnameToIPMap map[string]*IPInfo // key : interface name, value : IPInfo
+	HnsID         string
```

Comment:
@tamilmani1989 @rbtr 
I added the field to the existing endpoint state on CNS which is being used for Cilium as well. This should not break anything. But do we want to test cilium cni with this change on CNS?

Reply:
yes